pallets/pallet-did-lookup/src/migrations.rs

```diff
@@ -0,0 +1,90 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use crate::{Config, ConnectedAccounts, ConnectedDids, Pallet};
+use frame_support::{
+	dispatch::Weight,
+	traits::{Get, GetStorageVersion, OnRuntimeUpgrade},
+};
+use sp_std::marker::PhantomData;
+
+pub struct LookupReverseIndexMigration<T>(PhantomData<T>);
```

Comment:
It is not really necessary as the origin was blocked and there cannot be any linked account. Should I remove it altogether?